relm-core/examples/main.rs

```diff
@@ -68,65 +76,89 @@ fn main() {
     futures_glib::init();
     gtk::init().unwrap();
 
+    // This is a layout container that will stack widgets from the top down
```

Comment:
"top down" is not exactly true because we could use [`pack_end()`](http://gtk-rs.org/docs/gtk/trait.BoxExt.html#tymethod.pack_end).